extensions/amp-viz-vega/amp-viz-vega.md

```diff
@@ -28,14 +28,14 @@ Displays visualizations created using <a href="https://vega.github.io/vega/">Veg
 <table>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a></td>
+    <td><a href="https://amp.dev/documentation/guides-and-tutorials/learn/experimental.html">Experimental</a></td>
```

Comment:
Use the canonical URL https://amp.dev/documentation/guides-and-tutorials/learn/experimental

Reply:
fixed